HW2_Kravchenko/H2_calc2.py

```diff
@@ -0,0 +1,48 @@
+def divide(a, b):
+    """Функция для деления"""
+    if b == 0:
+        return "Ошибка: Деление на ноль!"
+    return a / b
+
+def p_multiply (a, b):
+    """Функция умножения"""
+    return(a * b)
+
+def sum_lesha(a, b):
+    """Функция сложения"""
+    return(a + b)
+def minus(a, b) :
+    """Функция вычитания"""
+    return(a-b)
```

Comment:
операнды в математических операциях принято разделять запятыми
```suggestion
    return(a - b)
```